src/dom/nativescript-vue-next/runtime/registry.ts

```diff
@@ -197,6 +199,10 @@ export function registerNativeElements() {
         'spinBox',
         () => new RNSpinBox(),
     )
+    registerElement<RNSvg, SvgProps>(
+        'svg',
```

Comment:
I'm unsure how the Svelte compiler will handle the tag name `svg` – I'm worried that it might special-case it. Have you tried running the code and checking what the Intellisense looks like when you try to write attributes on an `<svg>` element? If the suggested attributes appear to be inferred from the HTML SVG element's implementation, we may have to rename this to `<qtsvg>` or something to avoid the name clash. If so, we'd also have to make a corresponding change in [src/svelte-nodegui.ts](/src/svelte-nodegui.ts).